### ISSUE TITLE
Create firebase database for storing anomalies

### ISSUE THREAD

**Issue by jvikstrom** [Wire-Flies/MVK-SAAB#6]

**Comment by AntonIvarsson** [comment]
Kan börja rita lite på vilket json schema vi ska ha! Men kmr ju behöva ändras eftersom vi inte vet exakt vad en anomali är haha

**Comment by shrew97** [comment]
Jag skulle gärna vilja vara med i json-schema diskussionen, för att datan som går via firebase måste vi kunna visualisera i Cesium också :D Har börjat få lite koll på vad biblioteket klarar av när det gäller sånt 👍

**Comment by AntonIvarsson** [comment]
Absolut! Toppen! Känns som bra sätt att börja på, göra ett ungefärligt schema och sen köra in lite fake anomalier som du kan testa med :)

**Comment by jvikstrom** [comment]
Tror att Simon sa att ain kommer i princip bara säga om ett flygs nuvarande position är en anomali eller inte. Stämmer det @simonakesson?

**Comment by AntonIvarsson** [comment]
###   _**Okej! Ja då blir det ju lätt att göra schema @jvikstrom**_

**Comment by simake** [comment]
Ja, precis! Se det som en svart låda som tar en position (och tillhörande attribut såsom fart och riktning) och spottar ut true eller false!

**Comment by shrew97** [comment]
Yes, sedan måste vi få tag på plan-information som gjorde att den spottade ut true så att vi kan visualisera den :+1: Det schemat får vi lösa @AntonIvarsson, så vi är överens vad som bör gå genom firebase för att göra det niiiice :D

**Comment by AntonIvarsson** [comment]
Alright, men jag fixar så vi allt innan schemat med firebase så kollar vi på det tillsammans!

**Comment by AntonIvarsson** [comment]
Fixat basic kod för att sätta upp och logga in i firebase, men verkar inte kunna pusha(ny branch såklart) någon som har provat? eller haft problem med fler ssh nycklar?

**Comment by jvikstrom** [comment]
@AntonIvarsson Vad är det för error du får? Du borde kunna pusha tror jag

**Comment by AntonIvarsson** [comment]
git@github.com: Permission denied (publickey).

brukar vara när det är nåt lattjo med ssh väl? ska dubbelkolla min connection

**Comment by AntonIvarsson** [comment]
Löst. Om någon annan får problem med flera nycklar, min config nu:

Host gits-15.sys.kth.se-aivarss
	HostName gits-15.sys.kth.se
	User git
	IdentityFile ~/.ssh/id_rsa


Host github.com-AntonIvarsson
	HostName github.com
	User git
	IdentityFile ~/.ssh/id_rsa_AntonIvarsson

**Comment by AntonIvarsson** [comment]
Rättså klart ändå?
Ska fixa inläsning av cvs från servern, men datalagringen är klar
